tools/SetupFlow/DevHome.SetupFlow/Models/CloningInformation.cs

```diff
@@ -234,7 +234,15 @@ public string RepositoryProviderDisplayName
             {
                 try
                 {
-                    _repositoryProviderDisplayName = RepositoryProvider.DisplayName;
+                    // This can be null in the case of URL cloning.
+                    if (RepositoryProvider == null)
+                    {
+                        _repositoryProviderDisplayName = "git";
+                    }
+                    else
+                    {
+                        _repositoryProviderDisplayName = RepositoryProvider.DisplayName;
+                    }

```

Comment:
nit: Can be simplified
```suggestion
                    // This can be null in the case of URL cloning.
                    _repositoryProviderDisplayName = RepositoryProvider?.DisplayName ?? "git";
```